CONTRIBUTING.md

```diff
@@ -127,27 +127,82 @@ mvn test
 Integration tests are also written using Junit4. They coexist with the unit tests in the `src/test`
 subdirectory. Integration tests follow the naming convention `*IT.java` (e.g. `DataTestIT.java`),
 which enables the Maven Surefire and Failsafe plugins to differentiate between the two types of
-tests. Integration tests are executed against a real life Firebase project, and therefore
-requires an Internet connection.
-
-Create a new project in the [Firebase console](https://console.firebase.google.com/) if you do
-not already have one. Use a separate, dedicated project for integration tests since the test suite
-makes a large number of writes to the Firebase realtime database. Download the service account
-private key from the "Settings > Service Accounts" page of the project, and save it as
-`integration_cert.json` at the root of the codebase. Grant your service account the `Firebase
-Authentication Admin` role at
-[Google Cloud Platform Console / IAM & admin](https://console.cloud.google.com/iam-admin). This is
-required to ensure that exported user records contain the password hashes of the user accounts.
-Also obtain the web API key of the project from the "Settings > General" page, and save it as
-`integration_apikey.txt` at the root of the codebase.
-
-Some of the integration tests require an
-[Identity Platform](https://cloud.google.com/identity-platform/) project with multi-tenancy
-[enabled](https://cloud.google.com/identity-platform/docs/multi-tenancy-quickstart#enabling_multi-tenancy).
-An existing Firebase project can be upgraded to an Identity Platform project without losing any
-functionality via the
-[Identity Platform Marketplace Page](https://console.cloud.google.com/customer-identity). Note that
-charges may be incurred for active users beyond the Identity Platform free tier.
+tests.
+
+Integration tests are executed against a real life Firebase project. If you do not already
+have one suitable for running the tests against, you can create a new project in the
+[Firebase Console](https://console.firebase.google.com) following the setup guide below.
+Otherwise you can obtain the following credentials from your current project:
```

Comment:
Good point. Something like:

> If you already have a Firebase project, you'll need to obtain credentials to communicate and authorize access to your Firebase project.

Do you suggest adding the specific purpose in the sub points below? Something like:

> Service account certificate: This allows access to your Firebase project through a service account which is required for all integration tests. This can be downloaded...

> Web API key: This allows for Auth sign-in needed for some Authentication and Tenant Management integration tests. This is displayed...

Reply:
Great, added those in!